File: tests/human/x/clj/in_operator_extended.clj
(ns in-operator-extended
  (:require [clojure.string :as str]))

(def xs [1 2 3])
(def ys (filter odd? xs))
(println (boolean (some #{1} ys)))
(println (boolean (some #{2} ys)))

(def m {:a 1})
(println (contains? m :a))
(println (contains? m :b))

(def s "hello")
(println (str/includes? s "ell"))
(println (str/includes? s "foo"))

File: tests/human/x/fs/in_operator_extended.fs
let xs = [1;2;3]
let ys = xs |> List.filter (fun x -> x % 2 = 1)
printfn "%b" (List.contains 1 ys)
printfn "%b" (List.contains 2 ys)

let m = dict [("a",1)]
printfn "%b" (m.ContainsKey "a")
printfn "%b" (m.ContainsKey "b")

let s = "hello"
printfn "%b" (s.Contains "ell")
printfn "%b" (s.Contains "foo")
